File: Ohtap E Commerce Domain Models/Development/PHP/Objects/ProductSpecifications/ProductSpecification.php
<?php
namespace Ohtap\ECommerce\DomainModels\Objects\Promotions;


use Ohtap\ECommerce\DomainModels\Objects\Specifications\ISpecificationData;


class ProductSpecification  implements IProductSpecification
{

    public string $Description;
    public ISpecificationData $Specification;
    public string $Value;
    public int $State;
    public int $OrderNo;
    public string $Name;
    public string $ID;

}

File: Ohtap E Commerce Domain Models/Development/NetFramework/Models Objects/Serializable Objects/ProductSpecifications/ProductSpecification.cs
using Ohtap.ECommerce.Domain.Models.Serializable.Objects.LanguageDetails;
using Ohtap.ECommerce.Domain.Models.Serializable.Objects.Specifications;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using Ohtap.ECommerce.Domain.Models.Objects.ProductSpecifications;
using Ohtap.ECommerce.Domain.Models.Objects.Specifications;

namespace Ohtap.ECommerce.Domain.Models.Serializable.Objects.ProductSpecifications
{
    public class ProductSpecification : ProductSpecificationBasic, IProductSpecification
    {
        public ISpecificationData Specifications { get; set; }
    }
}
